options.go

```diff
@@ -41,6 +41,8 @@ type Options struct {
 
 	AuthenticatedEmailsFile  string   `flag:"authenticated-emails-file" cfg:"authenticated_emails_file" env:"OAUTH2_PROXY_AUTHENTICATED_EMAILS_FILE"`
 	AzureTenant              string   `flag:"azure-tenant" cfg:"azure_tenant" env:"OAUTH2_PROXY_AZURE_TENANT"`
+	BitbucketTeam            string   `flag:"bitbucket-team" cfg:"bitbucket_team" env:"OAUTH2_PROXY_BITBUCKET_TEAM"`
```

Comment:
Where does `BitbucketTeam` get read? Seems like there's a link missing between this line and `SetTeam()` being called. Did you test that setting the team works?

Reply:
I'm sorry. I've ported this PR from some code we had on a fork of the original `oauth2_proxy` from bitly.
Some bits got lost in translation. I've now fixed this and tested on a prod machine wrt teams and repos.